### ISSUE TITLE
Allow disabling recipe interpolation

### ISSUE THREAD

**Issue by FlatMapIO** [casey/just#735]
There are many tools that use go-template to get field values, such as
`kubectl get secret nfluxdb2-auth -n monitoring --template={{index .data "admin-password"}} | base64 -d`
which are difficult to render with the escept.

I tried `{{ '{{index .data "admin-password"}}' }}` 
and `"{{" '{{index .data "admin-password"}}' "}}"` 
that didn't work properly

**Comment by casey** [comment]
Did you get this working? 

I think you'll need another set of single quotes:

```
foo:
  kubectl get secret nfluxdb2-auth -n monitoring --template=--template='{{ '{{index .data "admin-password"}}' }}' | base64 -d
```

I think it would be good to be able to disable interpolation in individual recipes. It would probably need attributes, as proposed #604. It might be hard though, since the lexer would need to be know when it was lexing a recipe without interpolation.

**Comment by FlatMapIO** [comment]
I finally got the right result, but I think it's an inelegant solution, and in a complex scenario, the expression becomes extremely difficult to read.
```
kubectl get secret influxdb2-auth -n monitoring --template='{{'{{index .data "admin-password"}}'}}' | base64 -d
```

**Comment by casey** [comment]
I agree, it's definitely not very nice, and ideally could be improved.

**Comment by casey** [comment]
I just released v0.8.5, which allows escaping a `{{` by writing `{{{{`, so this should be much better now.
